osu.Game/Database/OsuDbContext.cs

```diff
@@ -111,10 +111,8 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             base.OnConfiguring(optionsBuilder);
             optionsBuilder
-                // this is required for the time being due to the way we are querying in places like BeatmapStore.
-                // if we ever move to having consumers file their own .Includes, or get eager loading support, this could be re-enabled.
-                .ConfigureWarnings(warnings => warnings.Ignore(CoreEventId.IncludeIgnoredWarning))
                 .UseSqlite(connectionString, sqliteOptions => sqliteOptions.CommandTimeout(10))
+                .ConfigureWarnings(w => w.Ignore(RelationalEventId.MultipleCollectionIncludeWarning))
```

Comment:
what's the reasoning for this warning ignore?

Reply:
It was added in order to suppress these notifications. Of course, it is a temporary solution and it should be reworked.
![osu_2021-03-14_16-32-32](https://user-images.githubusercontent.com/22450664/111078389-2b702580-8506-11eb-8b4e-4f51f77f41a1.jpg)

Reply:
i was hoping to get a more comprehensive answer to this, rather than "turning it on just to disable the warnings"...

sounds like it might be simpler/more proper to [just explicitly set the query splitting behaviour](https://docs.microsoft.com/en-us/ef/core/querying/single-split-queries#enabling-split-queries-globally) rather than just shutting up the warning. of course it has its ramifications. but i'm not sure whether the goal of this pull should be to just "make it work" or to actually straighten this out in the most proper way available.

Reply:
Specified `SingleQuery` for now since `SplitQuery` does not work well out of the box.

It generates the following messages:
2021-03-14 18:38:58 [important]: The query uses a row limiting operator ('Skip'/'Take') without an 'OrderBy' operator. This may lead to unpredictable results.

Here is one of the queries which causes this warning:
https://github.com/ppy/osu/blob/master/osu.Game/Skinning/SkinManager.cs#L166
Looks like we are not allowed to use `First`/`Last` methods without sorting.